### PR TITLE
Add a passing test suite for /api/articles/:article_id (comment_count)

### PR DESCRIPTION
**Pull request by stevenwu99**
I have completed FEATURE REQUES for /api/articles/:article_id  for Task 12 to caculate  comment_count.
Please check.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -62,7 +66,8 @@ describe ('GET /api/articles/:article_id',() => {
         expect(article).toHaveProperty("topic", expect.any(String));  
         expect(article).toHaveProperty("created_at");    
         expect(article).toHaveProperty("votes", expect.any(Number));   
-        expect(article).toHaveProperty("article_img_url", expect.any(String));   
+        expect(article).toHaveProperty("article_img_url", expect.any(String)); 
+        expect(article).toHaveProperty("comment_count", expect.any(Number));
```
Comment (Jodiech) [inline __tests__/app.test.js]: This should be a new test, not a refactor of the previous one ❗ Please add 400 and 404 tests too 👍

**Review by Jodiech — COMMENTED** [review]
Hey 👋 nice work on this ticket. I'd like you to create a new test for this, not refactor the previous ones! Please resubmit when you're done 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (stevenwu99) [reply __tests__/app.test.js]: I have add a new test for it

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the changes Steven! 👍

**Pull request merged**